### PR TITLE
ICMSLST-1446 Add licence reference to IMI detail view.

### PR DESCRIPTION
**Pull request by MattHolmes123**

### CODE REVIEW

File: web/templates/web/domains/case/manage/imi-case-detail.html [hunk 1]
```diff
@@ -21,16 +21,14 @@ <h3>IMI Summary</h3>
 
     {# TODO: Revisit when implementing ICMSLST-836 - Needs to be a download licence link #}
     {% set licence_download_url = "#" %}
-    {% set licence_hyperlink = "<a href='" + licence_download_url + "'>" + (app.licence_reference or "GBSIL9001730K") + "</a>" %}
+    {% set licence_hyperlink = "<a href='" + licence_download_url + "'>" + (licence_reference) + "</a>" %}
```
Comment (mikeylockwood) [inline web/templates/web/domains/case/manage/imi-case-detail.html:24]: `(licence_reference) == licence_reference`

**Review by mikeylockwood — APPROVED** [review]

File: web/templates/web/domains/case/manage/imi-case-detail.html (hunk 1 above)
Comment (MattHolmes123) [reply web/templates/web/domains/case/manage/imi-case-detail.html:24]: Ahh yer :) 
it was an or before.

File: web/domains/case/_import/views.py [hunk 2]
```diff
@@ -617,11 +617,19 @@ class IMICaseDetailView(PermissionRequiredMixin, LoginRequiredMixin, DetailView)
...
+        licence_doc = licence.document_references.filter(
+            document_type=CaseDocumentReference.Type.LICENCE
+        ).first()
+
         context = {
             "page_title": f"Case {self.object.get_reference()}",
             "case_type": "import",
             "contacts": self.object.importcontact_set.all(),
             "licence": self.object.get_most_recent_licence(),
+            "licence_reference": licence_doc and licence_doc.reference or "",
```
Comment (davidwtbuxton) [inline web/domains/case/_import/views.py:630]: This is a little hard to read, would be useful to have either parentheses to make the precedence explicit, or use an inline if.

**Review by davidwtbuxton — APPROVED** [review]

File: web/domains/case/_import/views.py (hunk 2 above)
Comment (MattHolmes123) [reply web/domains/case/_import/views.py:630]: I agree.
Please blame javascript for infecting my python brain.

**Pull request merged**